src/renderer.rs

```diff
@@ -328,24 +328,37 @@ fn entry_row(
             td {
                 p {
                     @if entry.is_dir() {
-                        a.directory href=(parametrized_link(&entry.link, sort_method, sort_order)) {
-                            (entry.name) "/"
-                            @if entry.is_symlink {
+                        @if entry.is_symlink {
+                            a.symlink href=(parametrized_link(&entry.link, sort_method, sort_order)) {
+                                (entry.name) "/"
                                 span.symlink-symbol { }
+                                @if let Some(symlink_dest) = entry.symlink_dest {
+                                    a.directory {(symlink_dest) "/"}
+                                }
+                            }
```

Comment:
```suggestion
                        @if let Some(symlink_dest) = entry.symlink_dest {
                            a.symlink href=(parametrized_link(&entry.link, sort_method, sort_order)) {
                                (entry.name) "/"
                                span.symlink-symbol { }
                                a.directory {(symlink_dest) "/"}
                            }
```

Reply:
If there is a command-line option to disable showing symlink destination, should we just hide symlink destination path or hide the symlink CSS and mark?

Reply:
I think this is reasonable, yes.

Reply:
> or hide the symlink CSS and mark?

I think it is okay to have dead CSS; only the destination and `symlink-symbol`.

Reply:
Changed in new commit.